samples/basic/servo_motor/dts/bindings/pwm-servo.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  PWM-driven servo motor. Example definition:
+
+  servo {
+    compatible = "servo";
```

Comment:
Incorrect compatible string.
But do we actually need an example here? We've got one in README.

Reply:
removed example as there's one on the README